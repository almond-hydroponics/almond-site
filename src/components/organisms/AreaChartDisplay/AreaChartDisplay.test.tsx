// react libraries
import { render } from '@testing-library/react';

// component
import AreaChardDisplay from './index';

describe('AreaChartDisplay component', () => {
	const props = {
		chartData: [15, 16, 20, 27, 21, 24, 21, 19, 16],
		chartColor: '#1967D2',
		backgroundColor: '#1967D2',
		labels: [
			'00:00',
			'03:00',
			'06:00',
			'09:00',
			'12:00',
			'15:00',
			'18:00',
			'21:00',
			'00:00',
		],
	};

	const { asFragment } = render(<AreaChardDisplay {...props} />);

	it('should render correctly', () => {
		expect(asFragment()).toMatchSnapshot();
	});
});
