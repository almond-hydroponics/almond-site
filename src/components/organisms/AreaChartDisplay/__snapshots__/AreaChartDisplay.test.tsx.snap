// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AreaChartDisplay component should render correctly 1`] = `
<DocumentFragment>
  <canvas
    height="150"
    width="300"
  />
</DocumentFragment>
`;
